.teamcity/settings.kts

```diff
@@ -1366,6 +1366,7 @@ object WPComPlugins_EditorToolKit : BuildType({
 
 				cd apps/editing-toolkit
 				yarn test:js --reporters=default --reporters=jest-junit --maxWorkers=${'$'}JEST_MAX_WORKERS
+				yarn lint:php
```

Comment:
I suggest moving it to a separate step (i.e. a new `script` block): the overhead is minimal and we'll get more info about which step failed and its average duration.